config/jobs-chromeos.yaml

```
@@ -364,6 +364,19 @@ _anchors:
         - ui.HotseatAnimation.shelf_with_navigation_widget_lacros
         - ui.WindowControl
 
+  v4l2-decoder-conformance-job: &v4l2-decoder-conformance-job
+    template: 'v4l2-decoder-conformance-template.jinja2'
+    kind: test
+    params: &v4l2-decoder-conformance-job-params
+      nfsroot: 'https://storage.staging.kernelci.org/rootfs/bookworm-gst-fluster/20240430.0/{debarch}/'
+      job_timeout: 30
+      videodec_parallel_jobs: '1'
```

```suggestion
      videodec_parallel_jobs: 1
```

No need for it to be a string, especially as you convert it to string in the template.